docs/user-guide/quick-start/admin-gui.md

```diff
@@ -5,78 +5,189 @@ As an Admin, you have the ability to create and deactivate users, change users'
+**2. Create a New Group:**
+- In the Groups window, click on the +New Group button at the top of the window.
+- Enter a name for the new group in the new window that appears.
+- Click "Create" to finalize the creation of the group.
+- A new window will be displayed showing the new group has been created. By default, the user who creates the new group is assigned as the Group Admin. It is important to notice that each group requires at least one group admin.
 
-![Create New Member](quick-start-images/add-new-member.png)
+
+![New Group](quick-start-images/14CreateNewGroup.png)
+***Create a new group**. Access the Groups section. A list of the available groups will be displayed. Click on **+New Group** to create a new group. Select a name for the new group, for example, **Demo ODM**. Click on Create. This action will create a new group named **Demo ODM**. The user who creates the group is automatically assigned as the Group Administrator.*
+
+Following these instructions will allow you to create a group using the GUI. There is no limit to the number of groups you can create, although it is not advisable to create multiple groups with the same name for clarity and management purposes.
+
+
+### Managing groups 
+To manage groups in the interface, navigate to the section Groups (instructions described above) and select the group you want to manage. 
+
+#### Add New Members:
+- Click on the +New members button to add members to the group.
+- In the new window, you can select members from the list or use the search bar to find specific users.
+
+![New Member](quick-start-images/15AddMembers.png)
+*Add new members to the group by clicking on **+New members**. A new window will appear where you can select users from the list or you can use the search bar to find specific users, e.g., **New User 2024***
+
+**Confirm New Members:**
+- After selecting the members, click **Add member to confirm**. The group will now display the recently added members.
+
+![Confirm New User](quick-start-images/17ConfirmMembers.png)
+*Confirm the new members to add by clicking **Add member**. The recently created group, **Demo ODM** for this example, will now show the recently added member, **New User 2024**.*
+
+
+#### Remove members
+To remove a member from a group, follow these steps:
+
+**1. Select the Group:**
+- Navigate to the group from which you intend to remove a user.
+- In the new window, either scroll through the list of members or use the search bar to find specific users.
+
+**2. Remove the Member:**
+- Click on the three dots next to the member's name and select **Remove member**.
+
+
+**Important Note**: Only members designated as **Group members** can be removed. If the user you intend to remove is an administrator, you must first ensure that another member is assigned as the group administrator, as each group must have at least one administrator. If you want to remove a user with the role of admin, you need to assign another administrator within the group.
```

Comment:
Admonitions syntax for notes and important information https://squidfunk.github.io/mkdocs-material/reference/admonitions/ Some examples here in User Guide
https://odm.demo.genestack.com/user-docs/user-guide/quick-start/admin-api/#use-case-example-delete-data-in-odm